vernac/ppvernac.ml

```diff
@@ -83,24 +88,30 @@ let pr_cumul_univdecl_instance l extensible =
 
 let pr_univdecl_constraints l extensible =
   if List.is_empty l && extensible then mt ()
-  else str"|" ++ spc () ++ prlist_with_sep (fun () -> strbrk" | ") pr_uconstraint l ++
+  else strbrk" | " ++ prlist_with_sep (fun () -> strbrk", ") pr_uconstraint l ++
```

Comment:
```suggestion
  else strbrk" | " ++ prlist_with_sep pr_comma pr_uconstraint l ++
```

Also, unless you insist on breaking before the pipe, you can use `pr_spcbar ()` in place of `strbrk" | "`.

Reply:
changed